pagespeed/envoy/envoy_cluster_manager.cc

```diff
@@ -0,0 +1,104 @@
+#include "envoy_cluster_manager.h"
```

Comment:
ASF license banner needs to be added in a couple of places.

Reply:
added ASF banner at all missing places.
code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/05e80e98faba31ff5a94f1244b1833fa88a1fae4